sentry.conf.py

```diff
@@ -276,12 +277,13 @@
 ENV_CONFIG_MAPPING = {
     'SENTRY_EMAIL_PASSWORD': 'mail.password',
     'SENTRY_EMAIL_USER': 'mail.username',
-    'SENTRY_EMAIL_PORT': ('mail.port', int),
-    'SENTRY_EMAIL_USE_TLS': ('mail.use-tls', bool),
+    'SENTRY_EMAIL_PORT': ('mail.port', Int),
```

Comment:
was this broken before then? i was re-implementing the type casting that was present

Reply:
The `Int` one wasn't, but `Bool` was. I just did `Int` to do them both the same. Coersion happens by default in `env()` when y ou pass a default. And these were doing like, `env('SENTRY_EMAIL_USE_TLS', False)` which inferred the type as Bool, to do the correct coersion.

The problem here with lowercase `bool` is that it doesn't make a lot of sense for env vars. When converting a string to a bool, `bool('0')` for example, is True. Whereas my `Bool('0')` is False. Among other expected values.

Reply:
:+1: